src/Sylius/Bundle/CoreBundle/Fixture/OrderFixture.php

```diff
@@ -175,13 +175,16 @@ private function generateItems(OrderInterface $order): void
     {
         $numberOfItems = random_int(1, 5);
         $products = $this->productRepository->findAll();
+        $shippingMethods = $this->shippingMethodRepository->findAll();
+        $shippingMethodsAvailable = (bool) count($shippingMethods);
```

Comment:
```suggestion
        $shippingMethodsAvailable = count($shippingMethods) > 0;
```

Reply:
Or we can even skip this variable

Reply:
I want to keep it as I don't want to count n times the same result set in the for loop. :)